apis/core/v1beta1/openstackcontrolplane_types.go

```diff
@@ -262,11 +262,13 @@ type RabbitmqSection struct {
 // MemcachedSection defines the desired state of Memcached services
 type MemcachedSection struct {
 	// +kubebuilder:validation:Optional
-	// +kubebuilder:default=false
+	// +kubebuilder:default=true
+	// +operator-sdk:csv:customresourcedefinitions:type=spec,xDescriptors={"urn:alm:descriptor:com.tectonic.ui:booleanSwitch"}
```

Comment:
I'm curious as to what this annotation is doing.  I haven't seen it before.  Is it something add by a newer version of `operator-sdk`?

Reply:
IIUC these are code markers supported by operator-sdk. While the existing annotations work for crd, these work for csv. https://v0-19-x.sdk.operatorframework.io/docs/golang/legacy/references/markers/

These were previously added to the other part of specs so I'm adding the same annotations for consistency. I noticed I didn't regenerate csv properly after making the change, which is not fixed in the latest change.

If we want I can make it a separate change.

Reply:
Cool, I was just wondering.  I think it's fine to have it in this PR.